lib/capybara/selenium/driver.rb

```diff
@@ -455,6 +459,10 @@ def find_context
     browser
   end
 
+  def active_element
```

Comment:
Would this be a good method to add to the `Session` public API?

Reply:
I don't think so at the moment